### PR TITLE
[WIP] Add Disability @ RustFest blog post

### PR DESCRIPTION
**Pull request by skade**
This still requires some links to be filled in and maybe some
wordsmithing.

### CODE REVIEW

File: _posts/2018-08-30-disability-at-rustfest.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+title: "Disability @ RustFest"
+#socialImageSrc: "/assets/posts/rome.jpg"
+authors:
+  - skade
+  - katsel
+---
+
+RustFest Rome has a special program for disabled Rustaceans. First of all, check our [accessibility statement](https://rome.rustfest.eu/accessibility/) for general information about the accessibility of our venue and our already present accessibility features, such as live transcription. If there is anything missing that you need, please get in touch with us as early as possible. Also, you may want to read about [Diversity @ RustFest](https://rome.rustfest.eu/diversity).
+
+Also, we have marked 5 of our free tickets explicitly for disabled people. Please go to [TODO: diversitytickets.org-link] to apply. "Disability" includes physical as well as mental disabilities. These tickets include entrance to the conference plus travel for you and a second person, should you require assistance. Please indicate on the application if this is the case. It won't influence your chances of acceptance, just quicken the process.
```
Comment (Hoverbear) [inline _posts/2018-08-30-disability-at-rustfest.md:12]: The `Also,` is unnecessary.

File: _posts/2018-08-30-disability-at-rustfest.md [hunk 2]
```diff
@@ -0,0 +1,20 @@
+---
+layout: post
+title: "Disability @ RustFest"
+#socialImageSrc: "/assets/posts/rome.jpg"
+authors:
+  - skade
+  - katsel
+---
+
+RustFest Rome has a special program for disabled Rustaceans. First of all, check our [accessibility statement](https://rome.rustfest.eu/accessibility/) for general information about the accessibility of our venue and our already present accessibility features, such as live transcription. If there is anything missing that you need, please get in touch with us as early as possible. Also, you may want to read about [Diversity @ RustFest](https://rome.rustfest.eu/diversity).
```
Comment (Hoverbear) [inline _posts/2018-08-30-disability-at-rustfest.md:10]: "RustFest Rome is proud to offer support for disabled Rustaceans."

Comment (katsel) [inline _posts/2018-08-30-disability-at-rustfest.md:10]: > First of all, check our…

First of all, please check our…

> Also, you may want to read about [Diversity @ RustFest](https://rome.rustfest.eu/diversity)

Also, you may want to read about [diversity at RustFest](https://rome.rustfest.eu/diversity)

File: _posts/2018-08-30-disability-at-rustfest.md (hunk 1 above)
Comment (katsel) [reply _posts/2018-08-30-disability-at-rustfest.md:12]: > Also, we have marked 5 of our free tickets explicitly for disabled people.

"We have marked 5 of our free diversity tickets explicitly for disabled people."

> "Disability" includes physical as well as mental disabilities.

"Disability includes physical as well as non-physical disabilities and neurodivergence."

File: _posts/2018-08-30-disability-at-rustfest.md [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+
+Also, we have marked 5 of our free tickets explicitly for disabled people. Please go to [TODO: diversitytickets.org-link] to apply. "Disability" includes physical as well as mental disabilities. These tickets include entrance to the conference plus travel for you and a second person, should you require assistance. Please indicate on the application if this is the case. It won't influence your chances of acceptance, just quicken the process.
+
+If you decide to attend on your own cost or your employers, please still let us know if we can help with anything.
```
Comment (katsel) [inline _posts/2018-08-30-disability-at-rustfest.md:14]: > your employers

"your employer's"

> if we can help with anything

"if we can assist with anything"

File: _posts/2018-08-30-disability-at-rustfest.md [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+
+## Support the program
+
+If you want to support this program, please head to the [tickets.rustfest.eu](https://tickets.rustfest.eu) and buy and additional supporter package.
```
Comment (katsel) [inline _posts/2018-08-30-disability-at-rustfest.md:18]: > If you want to

"If you would like to"

;)

> and buy and

"and buy an"

**Comment by apiraino** [comment]
@Hoverbear  @badboy maybe this, too, can be closed?

**Pull request closed without merging**